src/NumericalAlgorithms/LinearSolver/Convergence.hpp

```diff
@@ -0,0 +1,169 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines DataBox tags for the linear solver
+
+#pragma once
+
+#include <boost/none.hpp>
+#include <boost/optional.hpp>
+#include <cstddef>
+#include <iosfwd>
+
+#include "NumericalAlgorithms/LinearSolver/IterationId.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace LinearSolver {
+
+/*!
+ * \brief Criteria that determine the linear solve has converged
+ *
+ * \details The following criteria are implemented:
```

Comment:
Do you want to specify which one "wins" if multiple criteria are satisfied on the same iteration?